modules/passkey/contracts/SafeWebAuthnSignerProxyFactory.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import {ICustomECDSASignerProxyFactory} from "./interfaces/ICustomECDSASignerProxyFactory.sol";
+import {IP256Verifier} from "./interfaces/IP256Verifier.sol";
+import {ERC1271} from "./libraries/ERC1271.sol";
+import {WebAuthn} from "./libraries/WebAuthn.sol";
+import {SafeWebAuthnSignerProxy} from "./SafeWebAuthnSignerProxy.sol";
+/**
+ * @title SafeWebAuthnSignerProxyFactory
+ * @dev A factory contract for creating and managing WebAuthn proxy signers.
+ */
+contract SafeWebAuthnSignerProxyFactory is ICustomECDSASignerProxyFactory {
```

Comment:
This should not require an interface change. I think the singleton should be chosen per factory and set in the constructor.